cradlepoint/pages/api/editTest.js

```diff
@@ -0,0 +1,33 @@
+const { ObjectId } = require('mongodb');
+import connectToDb from "../../util/mongodb";
+import {testSchema} from "../../schemas/testSchema";
+/*
+  Edits the requested test from the database
+*/
+
+export default async (req, res) => {
+  if (req.method !== 'PUT') {
+    res.status(405).send({ message: 'Only PUT requests allowed' })
+  }
+  try{
+    const data = req.body;
+    const valid = await testSchema.isValid(data);
+    if (valid && ObjectId.isValid(data.testCaseId)){
+      const result = testSchema.cast(data);
+      // Set ID strings to Mongo ObjectId's
+      const newId = ObjectId(result._id);
+      const newTestCaseId = ObjectId(result.testCaseId);
```

Comment:
I think calling these new is misleading, since we aren't making any new ids. Can we call them something like just id and testCaseId?